### PR TITLE
Problem Size Option

### PR DESCRIPTION
**Pull request by MrBurmark**
Add problem size to kernels and add an option to run all problems with a specified problem size.
Currently this only prints the problem size in addition to iterations per rep, option for specific problem size coming later.

### CODE REVIEW

File: src/algorithm/SORTPAIRS.cpp
Comment (rhornung67) [inline src/algorithm/SORTPAIRS.cpp:43]: Is there a reason to not do the implementation of these methods in the header file? Less code to maintain.

File: src/polybench/POLYBENCH_2MM.cpp
Comment (rhornung67) [inline src/polybench/POLYBENCH_2MM.cpp:84]: So all the polybench kernels return zero in this PR? Is the idea that I come in after this is merged, adjust the polybench default kernel sizes to be ~1M for each and then return the correct value from getProblemSize()?

**Review by rhornung67 — COMMENTED** [review]
Some comments and questions.

File: src/algorithm/SORTPAIRS.cpp
Comment (MrBurmark) [reply src/algorithm/SORTPAIRS.cpp:43]: I see what you mean. I put them in the cpp files because the rest of the logic for problem sizes is in the source files.

File: src/polybench/POLYBENCH_2MM.cpp
Comment (MrBurmark) [reply src/polybench/POLYBENCH_2MM.cpp:84]: Ya whichever PR comes in second can fix up these values, I didn't want to redo any work you were doing with the polybench kernels.

File: src/algorithm/SORTPAIRS.cpp
Comment (rhornung67) [reply src/algorithm/SORTPAIRS.cpp:43]: I can see that perspective, but I don't think the extra code is worth it.

File: src/polybench/POLYBENCH_2MM.cpp
Comment (rhornung67) [reply src/polybench/POLYBENCH_2MM.cpp:84]: I'm going to wait until your stuff is merged because I don't want to deal with all the merge conflicts.

File: src/algorithm/SORTPAIRS.cpp
Comment (MrBurmark) [reply src/algorithm/SORTPAIRS.cpp:43]: Ok, Then I'll move this and iterations per rep into the headers.

**Review by rhornung67 — APPROVED** [review]

**Pull request merged**
